### DISCUSSION TITLE
Embed sur journal de mission

### DISCUSSION THREAD

**Discussion by ghost** [Crownicles/Crownicles#1778] [Nouvelles idées]
Mettre  le journal de mission (récapitulatif) dans un embed avec inline=False au lieu d'avoir tout sur une même ligne. Se serait plus joli et agréable à lire je pense.

**Comment by ghost** [comment]
![image](https://user-images.githubusercontent.com/122872130/216814615-7d72aba7-a7e1-4926-92f0-2d4b8bbfc666.png)
quelque chose qui ressemble à ça dans un embed

**Comment by KirIcare** [comment]
Je trouve ça bien plus clair et ergonomique, avec l'affichage actuel il m'arrive de me perdre et d'attribuer une valeur à la mauvaise ressource (avec un peu de bon sens, on se doute bien que rencontrer 2 personnes ne donne pas 68 gemmes mais 68 d'argent 🤣). Après a voir avec les contrib puisque c'est eux qui le feront si la proposition est acceptée

**Reply by ghost** [reply]
J"ai effacé mon ancien message, j'avais pas bien compris le sens de ta phrase. Sorry pour le double post dans le channel Suggestion.
